code/datums/wounds/_wounds.dm

```diff
@@ -51,8 +51,8 @@
 	var/list/treatable_by_grabbed
 	/// Tools with the specified tool flag will also be able to try directly treating this wound
 	var/treatable_tool
-	/// Set to TRUE if we don't give a shit about the patient's comfort and are allowed to just use any random sharp thing on this wound. Will require an aggressive grab or more to perform
-	var/treatable_sharp
+	/// Can we use a bandage/gauze on this wound in some kind of way?
+	var/accepts_gauze = TRUE
```

Comment:
Could you turn this and booleans in general into bitflags?
Even if there's no other booleans, just add it as such, as others may come down the line.
Stuff like `var/wound_flags = ACCEPTS_GAUZE` and `#define ACCEPTS_GAUZE (1<<0)`

Reply:
Reminder of this review

Reply:
woops, made it a flag, forgot to remove the var